pre-docs/writing_plans.md

```diff
@@ -527,3 +527,26 @@ plan pdb_discover {
   run_task('my_task', $nodes)
 }
 ```
+### Example plans
+
+Check out some example plans for inspiration writing your own.
+
+#### Facts Plans
+
+The [facts module](https://forge.puppet.com/puppetlabs/facts) contains tasks and plans to discover facts about target systems.
+
+The [facts](https://github.com/puppetlabs/puppetlabs-facts/blob/master/plans/init.pp) plan gathers facts using the facts task and sets the facts in inventory. 
+
+The [facts::info](https://github.com/puppetlabs/puppetlabs-facts/blob/master/plans/info.pp) plan utilizes the facts task to discover facts and map relevant fact values to targets.
+
+#### Reboot Plan
+
+The [reboot module](https://forge.puppet.com/puppetlabs/reboot) contains tasks and plans for managing system reboots.
+
+The [reboot](https://github.com/puppetlabs/puppetlabs-reboot/blob/master/plans/init.pp) plan restarts a target system and waits for it to become available again. This example shows an advanced use of plans.
```

Comment:
```suggestion
The [reboot plan](https://github.com/puppetlabs/puppetlabs-reboot/blob/master/plans/init.pp) restarts a target system and waits for it to become available again. This example shows an advanced use of plans.
```

Reply:
The 'advanced use' comment got me thinking: I wonder if we should group these by difficulty or what they demonstrate about Bolt (ie. example includes Apply from a plan) as categories, instead of what they do. I think people will be more interested in "I want an example of a super simple plan" or "I want to see how to get a list of hostnames from a result object" or something, rather than just what the plans do.

Reply:
That makes sense. updated